lapce-app/src/editor/view.rs

```diff
@@ -399,65 +509,113 @@ impl EditorView {
             },
         );
```

Comment:
I wasn't able to get debugging working in Lapce, so haven't tested that the line with the active breakpoint is rendered correctly.